x/incentives/keeper/distribute.go

```diff
@@ -799,7 +799,7 @@ func (k Keeper) Distribute(ctx sdk.Context, gauges []types.Gauge) (sdk.Coins, er
 
 	locksByDenomCache := make(map[string][]lockuptypes.PeriodLock)
 	totalDistributedCoins := sdk.NewCoins()
-	scratchSlice := make([]*lockuptypes.PeriodLock, 0, 10000)
+	scratchSlice := make([]*lockuptypes.PeriodLock, 0, 50000)
```

Comment:
Sorry, accidental drive by change, but should be benign as the scratch slice's need should be max_{gauges}(# accts gauge pays out to)